tests/pds/resiliency_tests.go

```diff
@@ -21,7 +21,7 @@ var _ = Describe("{RebootActiveNodeDuringDeployment}", func() {
 				Step("Start deployment, Reboot a node on which deployment is coming up and validate data service", func() {
```

Comment:
rename resiliency_tests.go file to resiliency_test.go

Reply:
Noted - This will go in the next PR of refining this framework